ui/eureka/zone_south_horn.ts

```diff
@@ -0,0 +1,797 @@
+import { EurekaZoneInfo } from './eureka';
+import southHornMap from './southhorn.png';
+
+// https://xivapi.com/search?indexes=Fate&filters=ID%3E=1717,ID%3C=1742&columns=Description,Name,Url
+
+export const zoneInfoSouthHorn: EurekaZoneInfo = {
+  mapImage: southHornMap,
+  mapWidth: 2048,
+  mapHeight: 2048,
+  shortName: 'southhorn',
+  hasTracker: false,
+  onlyShowInactiveWithExplicitRespawns: true,
+  treatNMsAsSkirmishes: true,
+  mapToPixelXScalar: 50,
+  mapToPixelXConstant: -50,
+  mapToPixelYScalar: 50,
+  mapToPixelYConstant: -50,
+  fieldNotes: [
+    {
+      id: 8,
+      name: {
+        en: 'Trade Tortoise',
+        de: 'Die Münzkröte',
+        fr: 'Tortue à pièces',
+        ja: 'コイントートス',
+        cn: '',
+        ko: '',
```

Comment:
Here (and all the other `LocaleText` properties defined in this file) the `cn`/`ko` keys should be excluded so that they're picked up by the missing translations tool.